cpp/src/replace/replace.cu

```diff
@@ -182,51 +192,107 @@ void replace_nulls_with_column(gdf_size_type size, Type* out_data, gdf_valid_typ
  *        `replace_nulls` with the apropiate data types.
  */
 /* ----------------------------------------------------------------------------*/
-struct replace_nulls_kernel_forwarder {
+struct replace_nulls_column_kernel_forwarder {
   template <typename col_type>
-  void operator()(gdf_size_type           nrows,
-                  gdf_size_type           new_values_length,
-                  void*            d_col_data,
-                  gdf_valid_type*  d_col_valid,
-                  const void*      d_new_value)
+  void operator()(gdf_size_type    nrows,
+                  void*            d_in_data,
+                  gdf_valid_type*  d_in_valid,
+                  const void*      d_replacement,
+                  void*            d_out_data)
   {
     const size_t grid_size = nrows / BLOCK_SIZE + (nrows % BLOCK_SIZE != 0);
```

Comment:
Use this instead: https://github.com/rapidsai/cudf/blob/branch-0.8/cpp/src/utilities/cuda_utils.hpp#L39